docs/cmd/kn_eventtype_create.md

```diff
@@ -21,9 +21,10 @@ kn eventtype create
 ### Options
 
 ```
-  -b, --broker string      Cloud Event broker
+  -b, --broker string      Cloud Event Broker. This flag is added for the convenience, since Eventing v1beta2 brokers as represented as KReference type.
```

Comment:
Still, I would remove the "is added for convenience" since this is an implementation detail that is confusing for a help page.

Reply:
I removed the whole sentence. Indeed it's an implementation detail.